src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+function makeString(number) {
+  const numbers = String(number).split('');
+
+  for (let i = numbers.length - 1, j = 1; i >= 0; i--, j++) {
+    if (j % 3 === 0 && i !== 0) {
+      numbers[i] = ',' + numbers[i];
+    }
+  }
+
+  return numbers.join('');
+}
+
+const population = document.querySelectorAll('.population');
+const total = [...population].map(item => {
```

Comment:
you don't have to make a copy here, map is non-mutating method

Reply:
But here we get nodelist, with the spread operator I made the array from the nodelist.